windows/generate_wazuh_msi.ps1

```diff
@@ -5,29 +5,35 @@ param (
     [string]$READY_TO_RELEASE = "",
     [string]$OPTIONAL_REVISION = "",
     [string]$SIGN = "",
+    [string]$WIX_TOOLS_PATH = "",
+    [string]$SIGN_TOOLS_PATH = "",
     [switch]$help
     )
 
-
 $MSI_NAME = ""
 $VERSION = ""
+$CANDLE_EXE = "candle.exe"
+$LIGHT_EXE = "light.exe"
+$SIGNTOOL_EXE = "signtool.exe"
 
 if(($help.isPresent)) {
     "
     This tool can be used to generate the Windows Wazuh agent msi package.
 
     PARAMETERS TO BUILD WAZUH-AGENT MSI:
-        1. READY_TO_RELEASE: yes or no.
-        2. OPTIONAL_REVISION: 1 or different
-        3. SIGN: yes or no.
+        1. OPTIONAL_REVISION: 1 or different
+        2. SIGN: yes or no.
+    OPTIONAL PARAMETERS:
+        3. WIX_TOOLS_PATH: Wix tools path
+        4. SIGN_TOOLS_PATH: sign tools path
 
     USAGE:
 
         * WAZUH:
-          $ ./generate_wazuh_msi.ps1 -READY_TO_RELEASE {{ yes|no }} -OPTIONAL_REVISION {{ BRANCH_TAG }} -SIGN {{ yes|no }}
+          $ ./generate_wazuh_msi.ps1  -OPTIONAL_REVISION {{ BRANCH_TAG }} -SIGN {{ yes|no }} -WIX_TOOLS_PATH {{ PATH }} -SIGN_TOOLS_PATH {{ PATH }}
```

Comment:
it isn't `BRANCH_TAG` is a `REVISION`

Reply:
Done in e2d87802c97c5651ba3806ef256bd2447f1fa26b